requirements-dev.txt

```diff
@@ -1,7 +1,8 @@
 jupyter
-matplotlib==2.0.0
+matplotlib>=2.0.2
 numpydoc
 openpyxl
+pandas
```

Comment:
Pandas again? Where?

Reply:
Yes, I believe it makes sense here, since these are the requirements for developers, who should take care not to break the pandas functionality.

Reply:
To prevent the case that pandas test cases are skipped during development. Gotcha.